microhapulator/workflows/analysis.smk

```diff
@@ -185,6 +113,28 @@ rule map_sort_and_index:
         """
 
 
+rule map_full_reference:
+    params:
+        refr=config["hg38path"],
```

Comment:
This was initially moved out of the main Snakemake file, and then moved back in...apparently not in the same spot!